app/utils/utils.ts

```diff
@@ -6,6 +6,10 @@ const downloadjs = require("downloadjs") as any;
  * White spaces are removed, "," replaced by "." is there is more than 1 dot or other chars NaN is returned.
  */
 export const parseStrToNumStrict = (source: string): number => {
+  if (source === null) {
```

Comment:
Is this possible that this will break forms already using this method? For example neumark estimation will be calculated for NaN for empty input field

Reply:
No if we would pass null value function would get null pointer as [few lines](https://github.com/Neufund/commit.neufund.org/blob/tomek/test_util_parseStrToNumStrict/app/utils/utils.ts#L17) below we call `source.replace`